parm/config/config.aero

```diff
@@ -0,0 +1,20 @@
+#!/bin/ksh -x
+#
+# UFS-Aerosols settings
+#
+# Directory containing GOCART configuration files. Defaults to parm/chem if unset.
+AERO_CONFIG_DIR=
+#
+# Path to the input data tree
+AERO_INPUTS_DIR=/scratch1/NCEPDEV/nems/Raffaele.Montuoro/data/NASA/ExtData
```

Comment:
This should point to a location in fix/. Once we have these files under the master fix location, sorc/link_workflow.sh will need to be updated to link the directory to the local fix. In the meantime, that can be done manually to the location in your personal space. Somehow, we already have both a fix/aer (for MERRA2 climo) and a fix/chem, so for now use fix/fix_aero_new as the root and we'll talk with @KateFriedman-NOAA very soon to figure this out.

For consistency with other variables, this would also preferably be named FIXaero.

So, this line would become `FIXaero=$HOMEgfs/fix/fix_aero_new`

And $HOMEgfs/fix/fix_aero_new would be a symlink to /scratch1/NCEPDEV/nems/Raffaele.Montuoro/data/NASA/ExtData you create manually for now.

Reply:
Added new fix file location in 0bed4636d89217657af2c11e793305ed82a597b3.

I would be in favor of retaining `AERO_INPUTS_DIR` for internal consistency in `config.aero`, but I'm fine with changing the variable name as suggested if needed.

Reply:
It's fine for now. May revisit it later.